### ISSUE TITLE
Claim.fee_address is missing

### ISSUE THREAD

**Issue by kauffj** [lbryio/chainquery#29]
Transaction [b5a1bf5345c2c727fb866bd312f6e6d23d607da139f54da8ad55ef2591f95cad](https://explorer.lbry.io/tx/b5a1bf5345c2c727fb866bd312f6e6d23d607da139f54da8ad55ef2591f95cad) contains a tip sent for [lbry://microsoft-releasing-linux-based#09e79ec21b8e331ddd12a6141fbf9037af9b0766](lbry://microsoft-releasing-linux-based#09e79ec21b8e331ddd12a6141fbf9037af9b0766).

However, this transaction is not reported as a tip in the SQL report that shows tips. I believe this because `claim.fee_address` is not set for claim_id `09e79ec21b8e331ddd12a6141fbf9037af9b0766`:

```
mysql> SELECT id, claim_id, fee_address FROM claim WHERE claim_id = '0f399396ec18a783f9bc66d58fe2095b31f36d40';
+--------+------------------------------------------+-------------+
| id     | claim_id                                 | fee_address |
+--------+------------------------------------------+-------------+
| 133791 | 0f399396ec18a783f9bc66d58fe2095b31f36d40 |             |
+--------+------------------------------------------+-------------+
```

**Comment by kauffj** [comment]
(It may be appropriate that `fee_address` is blank here and we need to revise the approach for determining tips. @tiger5226 and I have been discussing over Slack.)

**Comment by tiger5226** [comment]
claim[0f399396ec18a783f9bc66d58fe2095b31f36d40](https://chainquery.lbry.io/api/sql?query=SELECT%20*%20FROM%20claim%20WHERE%20claim_id%20=%20%2209e79ec21b8e331ddd12a6141fbf9037af9b0766%22), does not have a a fee address, therefore it cannot be tipped. The app from what I understand uses the fee_address to tip. In the case you provided, this is a support instead of a tip. The claim owner cannot unlock the tip. If you removed the `INNER JOIN` you will see the support in chainquery. 

```SQL
SELECT support.supported_claim_id, support.transaction_hash
FROM support
LEFT JOIN claim ON claim.claim_id = support.supported_claim_id
LEFT JOIN address ON address.address = claim.fee_address
LEFT JOIN transaction ON transaction.hash = support.transaction_hash
-- INNER JOIN transaction_address ON transaction_address.transaction_id = transaction.id AND transaction_address.address_id = address.id
WHERE support.transaction_hash = "b5a1bf5345c2c727fb866bd312f6e6d23d607da139f54da8ad55ef2591f95cad"
```

From what I understood from the tip tracking issue is that @lyoshenka did not want to include supports where it could be taken away but only tips that the claim owner can unlock. We can get both supports and tips in chainquery.

Again my understanding could be wrong. Happy to be corrected here and I can revisit the query.

**Comment by tiger5226** [comment]
Talked with @kauffj and he agrees it is not a bug. Also https://lbryio.github.io/lbry/#wallet_send says 

>>If given a claim id, send a tip
to the owner of a claim specified by uri. A tip is a claim support where the recipient
of the support is the claim address for the claim being supported.

Also attempting to tip a claim that has no fee_address will result in daemon error.

**Comment by tiger5226** [comment]
This was a misunderstanding. When a claim is supported or tipped the address of the output that created/updated the claim is where the LBC goes. 

- [ ] Add `claim_address` field to claim.
- [ ] Add an upgrade script to set the `claim_address` field for all claims.

**Comment by tiger5226** [comment]
Closed out @ 8f4959964e6a2a7a50f43cb9b3b8d12a4a4ddbcf
